src/fides/api/ctl/routes/datamap.py

```diff
@@ -113,7 +117,7 @@ async def export_datamap(
     # load resources from server, filtered by organization
     try:
         try:
-            organization = await get_resource(
+            organization = await get_resource_with_custom_fields(
```

Comment:
Initial thoughts around that test failure are that maybe we are looking for custom fields on the wrong resource type (although I could be mistaken) - can custom fields go anywhere or focused on `System` currently?

Here they are specifically pulling in the `Organization` where a few lines below we pull in the remaining resource types (`System`, `Dataset`, etc.) - maybe if we use that query on those other objects it gets us what we are looking for?